### DISCUSSION TITLE
Organisation name

### DISCUSSION THREAD

**Discussion by bobturneruk** [AFM-SPM/home#3] [General]
AFM-Code has been suggested as a better name for this organisation. I suggest AFM-SPM might be even better. I suggest we agree on something and change it soon as it affects repository URLs.

**Comment by alicepyne** [comment]
Thanks @bobturneruk! I agree it would be good to sort soon. I'm still keen on AFMstats as its in all my preprints, and our manuscript, but we should be able to change the proofs/preprints if we definitely want to change. I think I prefer AFM-SPM to AFM-Code.

**Comment by paulmueller** [comment]
I'm fine with either of the two!

**Comment by bobturneruk** [comment]
I'd be happy to stay with afmstats. Could we maybe change the name, then make a new "afmstats" into which we fork "topostats" so Alice's references stay constant? What do you think @paulmueller? Will GitHub release "afmstats" so we could do this? Would this work for you @alicepyne? If we do change the name it seems from above we go with "AFM-SPM".

**Comment by paulmueller** [comment]
I think afmstats is not general enough. For me it implies software that does statistics with AFM data. But we really want something everyone can connect to (imaging, spectroscopy, file format readers, modeling, ...).

**Comment by alicepyne** [comment]
I'll let you know when've spoken to Joe later, but think it sounds easier to change to AFM-SPM!

**Comment by alicepyne** [comment]
Hi @bobturneruk and @paulmueller - spoke to @JoeBeton and we're happy to change to AFM-SPM and update the two papers in the accepted stage to reflect this!
